code/game/turfs/simulated/minerals.dm

```diff
@@ -815,3 +818,12 @@
 	smooth_icon = 'icons/turf/smoothrocks_hard.dmi'
 	hardness = 2
 
+/turf/closed/mineral/gibtonite/proc/defuses()
```

Comment:
merge this proc into defuse() and make defuse take a value force_perfect, don't need two procs that do the same thing